keyboards/planck/keymaps/pickle_jr/keymap.c

```diff
@@ -0,0 +1,196 @@
+#include "planck.h" 
+
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
+#define _SPECIAL 3
+#define _CALC 4
+#define _NAV 5
+#define _ADJUST 6
+
+#define LOWER MO(_LOWER)
+#define RAISE MO(_RAISE)
+
+#define LPSFT SFT_T(KC_RPRN)
+#define LCCTL CTL_T(KC_LCBR)
+#define RCCTL MT(MOD_RCTL,KC_RCBR)
+#define RPSFT MT(MOD_RSFT,KC_LPRN)
+
+#define SCOLNAV LT(_NAV,KC_SCLN)
+#define TABCALC LT(_CALC, KC_TAB)
+#define QUOCALC LT(_CALC, KC_QUOT)
+#define LBRACK LT(_SPECIAL,KC_LBRC)
+#define RBRACK LT(_SPECIAL,KC_RBRC)
+
+#define LOCK RGUI(KC_L)
+
+enum custom_keycoedes {
+  CTLALTD = SAFE_RANGE,
+  VIMSAVE,
+  VIMWQ
+};
+
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  if(record->event.pressed) {
+    switch(keycode) {
+      case CTLALTD:
+        SEND_STRING(SS_DOWN(X_LCTRL)SS_DOWN(X_LALT)SS_DOWN(X_DELETE)SS_UP(X_DELETE)SS_UP(X_LALT)SS_UP(X_LCTRL));
+        return false;
+      case VIMSAVE:
+        SEND_STRING(":w");
+        return false;
+    }
+  }
+  return true;
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+/* Qwerty
+ * ,----------------------------------------------------------------------------------------------------.
+ * | Tab/CALC |   Q      |   W   |   E  |   R  |   T  |   Y  |   U  |   I  |   O   |   P      |  "/CALC |
+ * |----------+----------+-------+------+------+-------------+------+------+-------+----------+---------|
+ * | Esc      |   A      |   S   |   D  |   F  |   G  |   H  |   J  |   K  |   L   |   ;/Nav  | Enter   |
+ * |----------+----------+-------+------+------+------|------+------+------+-------+----------+---------|
+ * |   (/SFT  |   Z      |   X   |   C  |   V  |   B  |   N  |   M  |   ,  |   .   |   /      |  )/SFT  |
+ * |----------+----------+-------+------+------+------+------+------+------+-------+----------+---------|
+ * |   [/SPC  |   CTRL   |PrintS | GUI  | Bksp |Lower |Raise |  SPC | ALT  |VIMSAVE|  CTRL    |  ]/SPC  |
+ * `----------------------------------------------------------------------------------------------------'
+ */
+[_QWERTY] = {
```

Comment:
Could you reformat these to use this formatting:
https://github.com/qmk/qmk_firmware/blob/master/keyboards/planck/keymaps/default/keymap.c#L58